apps/financial-aid/web-osk/src/routes/status/FileUpload/fileUpload.tsx

```diff
@@ -40,11 +43,13 @@ const FileUpload = () => {
     CreateApplicationEventQuery,
   )
 
-  const sendFiles = async () => {
-    if (form?.otherFiles.length <= 0 || router.query.id === undefined) {
-      return
+  useEffect(() => {
+    if (error) {
+      setError(false)
     }
+  }, [form?.otherFiles])
```

Comment:
I could be wrong..... but is this needed? 😄

Reply:
No not really, but kinda weird that the error message is still showing when you have already uploaded a file? And if you cant upload file you have two error messages showing, I put this just in case, what do you think unnecessary?

Reply:
No, my bad, this makes sense 😃